packages/@okta/i18n/src/properties/login.properties

```diff
@@ -949,3 +959,6 @@ oie.consent.enduser.deny.label = No, it's not me
 oie.consent.enduser.title = Did you just try to sign in?
 oie.consent.enduser.email.allow.description = To continue, return to the device or window where you requested the email link. You may close this window at any time.
 oie.consent.enduser.deny.description = You may close this window at any time.
+
+# Browser thrown errors
+browser.error.NotAllowedError = The operation either timed out or was not allowed. See: https://www.w3.org/TR/webauthn-2/#sctn-privacy-considerations-client.
```

Comment:
If we are doing overriding we can give it a user friendly error message and not include links. 

We can just leave it as "The operation either timed out or was not allowed."